.github/dependabot.yml

```diff
@@ -1,15 +1,15 @@
 # To get started with Dependabot version updates, you'll need to specify which
 # package ecosystems to update and where the package manifests are located.
 # Please see the documentation for all configuration options:
-# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
```

Comment:
I was looking at this file because I realised that dependabot isn't keeping deps in the CNA template up-to-date. Just fixed the docs link and prettier formatted the file, I decided to leave it in this PR.

I'll look into enabling dependabot for this another time (it's not super urgent/important right now)